.changelog/17941.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+provider: Add `default_tags` argument (in public preview, see note above)
```

Comment:
It might be easiest to do the pull request work in #17974 since it is tightly coupled and so folks don't need to browse two change sets to see the initial implementation.